.circleci/config.yml

```diff
@@ -123,8 +123,9 @@ jobs:
       # The primary container is an instance of the first list image listed. Your build commands run in this container.
       - image: circleci/ruby:2.4
     working_directory: /tmp/deploy
+    parallelism: 5
```

Comment:
```suggestion
```

Parallelism in deploying the packages is already handled below with `parallel`.

Reply:
Thanks, fixed!